### PR TITLE
minor jest test improvements and one additional test on app.js

### PR DESCRIPTION
**Pull request by mhuckstepp**
# Description
Increases test coverage and test robustness. Also added comment on testing to the README

## Type of change
Testing

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge

# How Has This Been Tested?

- [ ] `npm test`

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/components/Stats/Stats.test.js [hunk 1]
```diff
@@ -10,6 +10,9 @@ afterEach(() => {
 describe('<Stats />', () => {
   test('Component renders', async () => {
     const { container } = await render(<Stats />);
+    console.log(container.firstChild);
```
Comment (xpeedy) [inline src/components/Stats/Stats.test.js]: is this console.log needed? just want to point out in case you missed it, but awesome job with the tests!

**Review by xpeedy — APPROVED** [review]
solid progress with the tests!

**Review by alexismarroquin7 — APPROVED** [review]
Changes are appropriately scoped and match the description.  Looks ready to start implementing more tests.

**Review by aleksandr-ardanov — APPROVED** [review]
Great job on tests, keep it up!

**Pull request merged**